micro-benchmarks/DRB159-gpu-missingtaskbarrier-orig-yes.c

```diff
@@ -0,0 +1,54 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+*/
+
+/*
+Nowait clause at line 30:58 will cause no implicit barrier after the tasks completion.
+Data Race Pairs, x[i]@36:9 and x[i]@36:20; x[i]@40:9 and x[i]@40:16
```

Comment:
similarly: we try to make sure there is only one pair of source locations causing data races. Can you check if this is possible?

Reply:
Sure thing.